lib/twitter_cldr/js/mustache/data_bundle.coffee

```diff
@@ -0,0 +1,38 @@
+###
+// Copyright 2012 Twitter, Inc
+// http://www.apache.org/licenses/LICENSE-2.0
+// TwitterCLDR (JavaScript) v{{version}}
+// Authors:     Cameron Dutro [@camertron]
+                Kirill Lashuk [@KL_7]
+                portions by Sven Fuchs [@svenfuchs]
+// Homepage:    https://twitter.com
+// Description: Provides date, time, number, and list formatting functionality for various Twitter-supported locales in Javascript.
+###
+
+{{#source_map}}
+###
+//@ sourceMappingURL={{locale}}.map
+###
+{{/source_map}}
+
+# required alias definition that adds twitter_cldr to Twitter's static build process
+###<<module_def>>###
+
+
+TwitterCldrDataBundle =
+{{{contents}}}
+
+root = if exports?
+  exports
+else
+  this.TwitterCldrDataBundle = {}
+  this.TwitterCldrDataBundle
+
+root[key] = obj for key, obj of TwitterCldrDataBundle
+
+if this.TwitterCldr?
+  try
```

Comment:
There is a certain song and dance about the `data` variable. This is because (if I understand correctly) of the funky scopes of variables here. As @camertron mentioned in the comment below, because of the enclosing closer that the code is wrapped in, the `data` variable available inside the closure is different from the one you get when you call `TwitterCldr.data`.

you could try loading these files in a browser or in node and try this:

```node

> var TwitterCldr = require('./lib/assets/javascripts/twitter_cldr/twitter_cldr.js');
> var data = require('./lib/assets/javascripts/twitter_cldr/en.js');
> TwitterCldr.set_data(data) 
> TwitterCldr.data
# undefined
> TwitterCldr.get_data()
# { ... actual data ... }
```

Now, you could try doing something like this 

```node
TwitterCldr.set_data = (bundle) ->
  @data = bundle
root['data'] = if this.TwitterCldrDataBundle? then this.TwitterCldrDataBundle? else null
```
(or something like that) to make the `data` variable public, but that will mess up scope inside other classes and if you try to call something like `TwitterCldr.NumberFormatter.data()`, that will raise the `Data not set` exception (as things stand)

Unless I'm understanding something incorrectly, we can't really access `TwitterCldr.data` directly but through the accessor method, which, as things stand, raises an exception when the data is not set.

(I don't know if I explained the problem with sufficient clarity. If so, let me know and I'll make a branch to demonstrate)